packages/devtools/lib/src/memory/memory_data_view.dart

```diff
@@ -116,29 +117,26 @@ class MemoryDataChildProvider extends ChildProvider<BoundField> {
       switch (field.value.kind) {
         case InstanceKind.kPlainInstance:
           return instance.fields;
-          break;
         case InstanceKind.kList:
           final List<BoundField> result = [];
 
           int index = 0;
           for (dynamic value in instance.elements) {
-            result.add(new BoundField()
+            result.add(BoundField()
               ..decl = (FieldRef()..name = '[$index]')
               ..value = value);
             index++;
           }
           return result;
-          break;
         case InstanceKind.kMap:
           final List<BoundField> result = [];
           for (dynamic value in instance.associations) {
-            result.add(new BoundField()
+            result.add(BoundField()
               // TODO(terry): Need to handle nested objects for keys/values.
-              ..decl = (FieldRef()..name = '[${value.key.valueAsString}]')
+              ..decl = (FieldRef()..name = '\'${value.key.valueAsString}\'')
```

Comment:
nit: you could use double quotes here to make this a little more readable.
e.g.
"'${value.key.valueAsString}'"

Reply:
Then I'll get a linter warning to use single quotes.